### DISCUSSION TITLE
How install the blog plugin?

### DISCUSSION THREAD

**Discussion by pivic** [squidfunk/mkdocs-material#5333] [Q&A] [answered]
Hi!

I'm attempting to install and preview MkDocs insiders with the blog plugin.

I just cloned the mkdocs-material-insiders repo by using GitHub Desktop and then tried to use the blog plugin, but I see this error when I try to start the live preview server:

    mkdocs serve
    ERROR - Config value 'plugins': The "blog" plugin is not installed
    Aborted with 1 Configuration Errors!

I then tried cloning by using pip, but here's what happened:

***

pip install git+https://$[this_is_where_I_put_my_token,_without_brackets]@github.com/squidfunk/mkdocs-material-insiders.git

Defaulting to user installation because normal site-packages is not writeable
Collecting git+https://****@github.com/squidfunk/mkdocs-material-insiders.git
  Cloning https://****@github.com/squidfunk/mkdocs-material-insiders.git to /tmp/pip-req-build-j328tkuu
  Running command git clone --filter=blob:none --quiet 'https://****@github.com/squidfunk/mkdocs-material-insiders.git' /tmp/pip-req-build-j328tkuu
Password for 'https://github.com': 
  remote: Repository not found.
  fatal: Authentication failed for 'https://github.com/squidfunk/mkdocs-material-insiders.git/'
  error: subprocess-exited-with-error
  
  × git clone --filter=blob:none --quiet 'https://****@github.com/squidfunk/mkdocs-material-insiders.git' /tmp/pip-req-build-j328tkuu did not run successfully.
  │ exit code: 128
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× git clone --filter=blob:none --quiet 'https://****@github.com/squidfunk/mkdocs-material-insiders.git' /tmp/pip-req-build-j328tkuu did not run successfully.
│ exit code: 128
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

***

When asked to use a password, I both tried my GitHub password and my token; both produced the error message above.

When I do `pip list` I don't see `mkdocs-material-x.x.x+insiders-x.x.x`.

I'd appreciate any help with this.

**Comment by pivic** [comment, marked as answer]
Solved this myself by finding the answer in this thread: https://github.com/squidfunk/mkdocs-material/discussions/3816

In other words: I removed the `$` character and then I can clone away!

**Reply by squidfunk** [reply]
It's easier if you just put your token in the `GH_TOKEN` variable and then invoke the command as-is 😀

``` bash
export GH_TOKEN=<put-token-here>
pip install git+https://${GH_TOKEN}@github.com/squidfunk/mkdocs-material-insiders.git
```
